packages/dashboard/src/components/pages/platforms/components/instruments/instruments.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { TStringTranslation } from 'Types'
+
+const Instruments: React.FC<IInstrumentsType> = ({ title, active_symbols = [] }) => {
+    return (
+        <div className='dw-instruments'>
+            <Text className='dw-instruments__title' size='m' weight='bold'>
+                {title}
+            </Text>
+            {active_symbols.map((market, idx) => (
+                <React.Fragment key={idx}>
+                    <div className='dw-instruments__market'>
+                        <Icon className='dw-instruments__market-icon' icon={market.icon} width='48' height='48' />
+                        <div className='dw-instruments__market-title'>
+                            <Text size='sm' weight='bold'>
+                                {market.title}
+                            </Text>
+                            {market.description && <Text>{market.description}</Text>}
+                        </div>
+                    </div>
+                    {market.submarkets.map((submarket, idy) => (
+                        <div key={idy} className='dw-instruments__submarket'>
+                            <Text className='dw-instruments__submarket-title' size='sm' weight='bold'>
+                                {submarket.title}
+                            </Text>
+                            {submarket.description && (
+                                <Text className='dw-instruments__submarket-subtitle'>{submarket.description}</Text>
+                            )}
+
+                            <div className='dw-instruments__symbol'>
+                                {submarket.symbols.map((symbol, idz) => (
```

Comment:
`idz` 😬 

Does `symbol` have a unique identifier? Please use that instead.

Reply:
no they don't 😄 , its only for react render key usage, for clarity, i can change it to symbol_index?

Reply:
The issue is that we shouldn't use indexes as React keys. If you can somehow create a unique key (e.g. by using the symbol name in combination with the submarket name) use that over the index. Indexes are a last (bad) resort.

Reply:
thanks, thats really good article, will do as suggested 👍